frontend/components/board.js

```diff
@@ -0,0 +1,167 @@
+var React = require('react')
+
+function Square(props) {
+  return (
+    <button className="square" id={props.value}>
+    </button>
+  );
+}
+
+module.exports = React.createClass({
+  getInitialState: function() {
+    var squares = Array(100).fill(null);
+    var isBlack = true;
+    for(var i=0; i<squares.length; i++) {
+      if(i%10 == 0) {
```

Comment:
So are we going for every other row being alternating colors? Idk Langton's very well to say if it matters or not.

Reply:
I think one of the early posts on Langton's Ant I'd looked at had mentioned beginning the game with a checkerboard, but most of what I'm seeing now says to start out with a board of all white squares. I'll update that in a followup commit as that seems to be the more accurate approach